db/tables.py

```diff
@@ -213,12 +213,119 @@ def merge_tables(
             select(merged_columns, distinct=True).select_from(merge_join)
         )
         conn.execute(insert_stmt)
-        if drop_original_tables:
-            table_one.drop()
-            table_two.drop()
+
+    if drop_original_tables:
+        if table_one.foreign_keys:
+            table_one.drop(bind=engine)
+            table_two.drop(bind=engine)
+        else:
+            table_two.drop(bind=engine)
+            table_one.drop(bind=engine)
+
     return merged_table
 
 
+def move_column_between_related_tables(
+        source_table_name,
```

Comment:
We might need to change the function signature later, since we will already have reflected table objects. I don't think that needs to be fixed in this PR, though, we can update it when we actually use the function.

Reply:
I actually think we should go rather the other way.  These sorts of DDL functions will be safest, and be easiest to chain together if they take only strings (or lists of strings) to refer to tables, columns, schemas, etc.  I mostly actually regret returning the actual table object,  since it'll make these harder to compose.